code/modules/mob/living/emote.dm

```diff
@@ -262,6 +285,8 @@
 				return 'sound/voice/human/womanlaugh.ogg'
 			else
 				return pick('sound/voice/human/manlaugh1.ogg', 'sound/voice/human/manlaugh2.ogg')
+		if(H.dna.species.id == SPECIES_MOTH && (!H.mind || !H.mind.miming))
+			return 'sound/emotes/mothlaugh.ogg'
```

Comment:
Having just said that of course, this one is checking `H.mind` and `H.mind.miming` twice in a row, so maybe it'd be worth changing *one* of them:
```dm
/datum/emote/living/laugh/get_sound(mob/living/user)
	if(!ishuman(user) || user.mind?.miming)
		return null

	var/mob/living/carbon/human/H = user
	if(H.dna.species.id == SPECIES_HUMAN)
		if(user.gender == FEMALE)
			return 'sound/voice/human/womanlaugh.ogg'
		else
			return pick('sound/voice/human/manlaugh1.ogg', 'sound/voice/human/manlaugh2.ogg')
	if(H.dna.species.id == SPECIES_MOTH)
		return 'sound/emotes/mothlaugh.ogg'
```